components/inputs/input-color.js

```diff
@@ -296,6 +296,10 @@ class InputColor extends InputInlineHelpMixin(PropertyRequiredMixin(FocusMixin(F
 			this.setFormValue(this.value);
 		}
 
+		if (changedProperties.has('readonly')) {
+			this.shadowRoot.querySelector('d2l-tooltip[for="opener"]')?.requestUpdate('for', '');
```

Comment:
Just to confirm I understand: So even these the `for` value of `id` hasn't changed, this is kind of faking the tooltip out by giving it a previous value of empty string so that it forces itself to find the new opener?

Wondering if it might be cleaner to be more explicit and use separate `opener-btn` and `opener-readonly` as the `id`s that we put on the opener? That way tooltip would trigger its own update.

Reply:
Yeah that is also what I was initially going to do but it was more lines of code and I figured I could get discussion started with this first, since not supporting this at all is also a reasonable take, I think.